app/src/organisms/SendProtocolToOT3Slideout/index.tsx

```diff
@@ -38,10 +38,23 @@ export function SendProtocolToOT3Slideout(
     srcFiles,
     mostRecentAnalysis,
   } = storedProtocolData
-  const { t } = useTranslation('protocol_details')
+  const { t } = useTranslation(['protocol_details', 'shared'])
 
   const [selectedRobot, setSelectedRobot] = React.useState<Robot | null>(null)
 
+  const isSelectedRobotOnWrongVersionOfSoftware = [
+    'upgrade',
+    'downgrade',
+  ].includes(
+    useSelector((state: State) => {
+      const value =
+        selectedRobot != null
+          ? getRobotUpdateDisplayInfo(state, selectedRobot.name)
+          : { autoUpdateAction: '' }
+      return value
```

Comment:
```suggestion
        return selectedRobot != null
          ? getRobotUpdateDisplayInfo(state, selectedRobot.name)
          : { autoUpdateAction: '' }
```

or 

`value` => `updateInfo` something (I don't have any good names  🤔 )